indico/modules/designer/templates/template.html

```diff
@@ -1,268 +1,312 @@
-{% extends 'categories/management/base.html' if category else 'events/management/base.html' %}
+{% extends 'categories/management/base.html' if category else 'events/management/full_width_base.html' %}
```

Comment:
We want full-width in events but not in categories?
Otherwise consider adding a similar `full_width_base.html` template for categories too